mp/src/game/server/neo/neo_player.cpp

```diff
@@ -123,6 +123,7 @@ ConVar sv_neo_can_change_classes_anytime("sv_neo_can_change_classes_anytime", "0
 	true, 0.0f, true, 1.0f);
 ConVar sv_neo_change_suicide_player("sv_neo_change_suicide_player", "0", FCVAR_REPLICATED, "Kill the player if they change the team and they're alive.", true, 0.0f, true, 1.0f);
 ConVar sv_neo_change_threshold_interval("sv_neo_change_threshold_interval", "0.25", FCVAR_REPLICATED, "The interval threshold limit in seconds before the player is allowed to change team.", true, 0.0f, true, 1000.0f);
```

Comment:
I think I just had a stroke trying to read this

Reply:
Wrong line? Anyway assuming: `neo_sv_dm_max_class_dur` -> `neo_sv_dm_respawn_classpick_duration`?

Reply:
correct line, unrelated to this pr